tornado-drivers/opencl/src/main/java/uk/ac/manchester/tornado/drivers/opencl/runtime/OCLTornadoDevice.java

```diff
@@ -609,21 +610,22 @@ public int allocate(Object object, long batchSize, DeviceBufferState state) {
         if (buffer.getClass() == AtomicsBuffer.class) {
             state.setAtomicRegion();
         }
-        return -1;
+        return state.getXPUBuffer().size();
     }
 
     @Override
-    public synchronized int deallocate(DeviceBufferState deviceBufferState) {
+    public synchronized long deallocate(DeviceBufferState deviceBufferState) {
+        long memoryRegionDellocated = 0;
```

Comment:
refactor the variable to `deallocatedSpace` to keep consistency with the `allocate` method. Also, let's use the same variable names across all backends:
`PTXTornadoDevice`, `SPIRVTornadoDevice`.